src/cpp/session/modules/NotebookData.R

```diff
@@ -57,6 +57,11 @@
 
 .rs.addFunction("initDataCapture", function(outputFolder, options)
 {
+  pagedOption <- if (!is.null(options[["paged.print"]])) options[["paged.print"]] else getOption("paged.print")
```

Comment:
`options` here just has the chunk-specific options, not any of the default chunk options that might have been set in the setup chunk. I think we should respect the defaults set in the setup chunk for symmetry with the way we handle other options.

https://github.com/rstudio/rstudio/blob/5803d5b8e914516cdc27b92a89f6107a8895a50a/src/cpp/session/modules/rmarkdown/NotebookChunkOptions.cpp#L33-L44

https://github.com/rstudio/rstudio/blob/5803d5b8e914516cdc27b92a89f6107a8895a50a/src/cpp/session/modules/rmarkdown/NotebookExec.cpp#L232

Probably the most straightforward way to fix this is to just pass the merged chunk options when initializing the data capture module, although of course we should be sure that won't add values we meant to ignore!